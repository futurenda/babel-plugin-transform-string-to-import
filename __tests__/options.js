const pluginTester = require('babel-plugin-tester')
const plugin = require('../index.js')

pluginTester({
    plugin,
    tests: {
        'works with custom prefix': {
            code: `var a = '!iconName'`,
            pluginOptions: { prefix: '!' },
            snapshot: true
        }
    }
})