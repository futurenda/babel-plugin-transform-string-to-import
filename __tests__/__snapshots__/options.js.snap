// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform-string-to-import works with custom prefix: works with custom prefix 1`] = `
"
var a = '!iconName'

      ↓ ↓ ↓ ↓ ↓ ↓

import _iconName from 'iconName';
var a = _iconName;
"
`;
